docs/web-ui.md

```diff
@@ -119,14 +134,21 @@ Displays the following information:
 - History Archival: Whether History Archival is enabled
 - Visibility Archival: Whether Visibility Archival is enabled
 - Failover Version: Namespace Failover Version
-- Clusters: Cluster information
+- Clusters: Cluster information -->
+
+### Archive
+
+On self-hosted Temporal Clusters, Archive shows <a class="tdlp" href="/clusters#archival">Archived<span class="tdlpiw"><img src="/img/link-preview-icon.svg" alt="Link preview icon" /></span><span class="tdlpc"><span class="tdlppt">What is Archival?</span><br /><br /><span class="tdlppd">Archival is a feature that automatically backs up Event Histories from Temporal Cluster persistence to a custom blob store after the Closed Workflow Execution retention period is reached.</span><span class="tdlplm"><br /><br /><a class="tdlplma" href="/clusters#archival">Learn more</a></span></span></a> data of your Workflow Executions on the Namespace.
+
+You must have [Archival set up and configured](/cluster-deployment-guide#archival) in your self-hosted Cluster to see data.
 
-### Archival
+<!--AB: Needs some statement for Temporal Cloud users. Maybe:
```

Comment:
I'd leave this generic for now.  Temporal Cloud users can contact their Temporal representative for information about Archival on Temporal Cloud.

Reply:
Added: 
`For information and details on the Archive feature in Temporal Cloud, contact your Temporal representative.`